### PR TITLE
D-Kit : Update Localization_Configuration_cn.md and gnss_conf.pb.txt of the dev-kit

### PR DESCRIPTION
**Pull request by lishuanghua**
1、Update the configuration of the GNSS;
2、Modify the baud rate of the IMU;

### CODE REVIEW

File: docs/specs/D-kit/Waypoint_Following/Localization_Configuration_cn.md [hunk 1]
```diff
@@ -27,10 +27,11 @@
 ## 前提条件
 
  - 完成了[循迹搭建--车辆集成](Vehicle_Integration_cn.md)
+ - 购买了RTK基站账号
 
 ## 导航设备配置
 
-下面介绍了导航配置的方法。当设备正确接入系统后，在/dev/下面有名为ttyACM0的设备，即表示M2已经被正确的加载了。配置设备时，需要将设备的串口线连接上电脑的串口才可以对设备进行配置，也就是说，用来配置设备的电脑主机需要拥有串口。Windows下可以通过串口助手、串口猎人或者COMCenter等工具进行配置，Linux下可以通过Minicom、cutecom等工具进行配置。linux下建议使用cutecom软件，可使用`sudo apt-get install cutecom`来安装此软件，在终端中使用`sudo cutecom`命令打开该软件，在软件中`open`名为`ttyS0`的设备。
+下面介绍了导航配置的方法。当设备正确接入系统后，在/dev/下面有名为ttyACM0的设备，即表示M2已经被正确的加载了。配置设备时，需要将设备的串口线连接上电脑的串口才可以对设备进行配置，也就是说，用来配置设备的电脑主机需要拥有串口。Windows下可以通过串口助手、串口猎人或者COMCenter等工具进行配置，Linux下可以通过Minicom、cutecom等工具进行配置。linux下建议使用cutecom软件，可使用`sudo apt install cutecom`来安装此软件，在终端中使用`sudo cutecom`命令打开该软件，在软件中`open`名为`ttyS0`的设备。
```
Comment (storypku) [inline docs/specs/D-kit/Waypoint_Following/Localization_Configuration_cn.md:34]: [了]  as in [下面介绍了] should be removed. [的] as in  [M2已经被正确的加载] should be replaced with [地].

Comment (lishuanghua) [reply docs/specs/D-kit/Waypoint_Following/Localization_Configuration_cn.md:34]: OK, I will modify them right now.

File: docs/specs/D-kit/Waypoint_Following/Localization_Configuration_cn.md [hunk 2]
```diff
@@ -27,10 +27,11 @@
 ## 前提条件
 
  - 完成了[循迹搭建--车辆集成](Vehicle_Integration_cn.md)
+ - 购买了RTK基站账号
```
Comment (diegohu) [inline docs/specs/D-kit/Waypoint_Following/Localization_Configuration_cn.md:30]: “拥有RTK基站账号”

File: docs/specs/D-kit/Waypoint_Following/Localization_Configuration_cn.md [hunk 3]
```diff
@@ -77,7 +78,18 @@ $cmd,set,netipport,111,112,113,114,8000*ff
 $cmd,set,netuser,username:password*ff
 $cmd,set,mountpoint,XMJL*ff
 ```
-这里我们假设您所使用的无线路由器的IP地址为192.168.0.1,那么我们将M2主机的IP地址设置为192.168.0.123，子网掩码为255.255.255.0，网关为192.168.0.1，netipport设置的是RTK基站的IP地址和端口，netuser设置的是RTK基站的用户名和密码，mountpoint是RTK基站的挂载点。网络配置请依据自己所使用的路由器的实际情况自行更改为相应的配置，RTK基站信息请以自己的实际情况为准。注意：在M2的网络模块配置完成后，在IPC主机中应该是可以ping通IMU的ip地址的；否则，IMU无法正常联网，在后续的GNSS信号检查中会一直显示SINGLE而不是我们期望的NARROW_INT。
+
+假如您购买了一个RTK基站账号，账号信息如下：
```
Comment (diegohu) [inline docs/specs/D-kit/Waypoint_Following/Localization_Configuration_cn.md:82]: 假如您通过自建RTK基站或者购买RTK基站服务的方式拥有了一个RTK基站账号，账号信息如下：

**Pull request merged**